Defra.Cdp.Backend.Api/Endpoints/MigrationEndpoints.cs

```diff
@@ -0,0 +1,93 @@
+using Defra.Cdp.Backend.Api.Services.Github;
+using Defra.Cdp.Backend.Api.Services.Migrations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Defra.Cdp.Backend.Api.Endpoints;
+
+public static class MigrationEndpoints
+{
+    public static void MapMigrationEndpoints(this IEndpointRouteBuilder app)
+    {
+        app.MapGet("/migrations/available", ListServicesWithMigrations);
+        app.MapGet("/migrations/available/{service}", ListAvailableMigrationsForService);
+        app.MapGet("/migrations/run/{id}", FindMigrationById);
+        app.MapGet("/migrations/run", SearchMigrations);
```

Comment:
migrations/run => SearchMigrations, doesnt read write to me :-)

Reply:
happy with either, whichever one reads better